src/main/java/com/questhelper/helpers/quests/theheartofdarkness/TheHeartOfDarkness.java

```diff
@@ -0,0 +1,1028 @@
+                "the first lever.");
+        slideAlongIceLedge = new ObjectStep(this, ObjectID.ICY_LEDGE, new WorldPoint(1714, 9632, 1), "Climb across the ice ledge to the north. Avoid the " +
+                "floor spike traps on the way.");
+        slideAlongIceLedge.setLinePoints(List.of(
+                new WorldPoint(1707, 9607, 1),
+                new WorldPoint(1719, 9607, 1),
+                new WorldPoint(1715, 9615, 1),
+                new WorldPoint(1716, 9619, 1),
+                new WorldPoint(1716, 9629, 1)
+        ));
+
+        slideAlongIceLedgeBackToSecondLever = new ObjectStep(this, ObjectID.ICY_LEDGE_54534, new WorldPoint(1714, 9641, 1), "Climb back across the icy ledge.");
+        climbUpLedgeToSecondLever = new ObjectStep(this, ObjectID.ICE_54542, new WorldPoint(1684, 9659, 0), "Climb back up to pull a lever you missed.");
+        pullFirstLever.addSubSteps(slideAlongIceLedgeBackToSecondLever, climbUpLedgeToFirstLever, climbUpLedgeToSecondLever);
+        pullSecondLever = new ObjectStep(this, NullObjectID.NULL_55368, new WorldPoint(1711, 9660, 1), "Pull the lever in the north-east room. Avoid the " +
+                "floor traps.");
+        pullSecondLever.setLinePoints(List.of(
+                new WorldPoint(1710, 9648, 1),
+                new WorldPoint(1710, 9653, 1),
+                new WorldPoint(1713, 9655, 1),
+                new WorldPoint(1713, 9658, 1),
+                new WorldPoint(1711, 9660, 1)
+        ));
+        jumpOverFrozenPlatforms = new ObjectStep(this, ObjectID.LEDGE_54541, new WorldPoint(1686, 9659, 1), "Jump over the frozen platforms to the " +
+                "north-west, and climb down the ledge there.");
+        jumpOverFrozenPlatforms.setLinePoints(List.of(
+                new WorldPoint(1711, 9660, 1),
+                new WorldPoint(1713, 9658, 1),
+                new WorldPoint(1713, 9655, 1),
+                new WorldPoint(1710, 9653, 1),
+                new WorldPoint(1710, 9648, 1),
+                new WorldPoint(1703, 9653, 1),
+                new WorldPoint(1698, 9655, 1),
+                new WorldPoint(1694, 9655, 1),
+                new WorldPoint(1694, 9659, 1),
+                new WorldPoint(1691, 9659, 1),
+                new WorldPoint(1686, 9659, 1)
+        ));
+
+        pullThirdLever = new ObjectStep(this, NullObjectID.NULL_55366, new WorldPoint(1671, 9661, 0), "Pull the third lever in the north-west room.");
+        pullThirdLever.setLinePoints(List.of(
+                new WorldPoint(1681, 9656, 0),
+                new WorldPoint(1677, 9653, 0),
+                new WorldPoint(1668, 9653, 0),
+                new WorldPoint(1668, 9656, 0),
+                new WorldPoint(1671, 9659, 0),
+                new WorldPoint(1671, 9661, 0)
+        ));
+        pullFourthLever = new ObjectStep(this, NullObjectID.NULL_55365, new WorldPoint(1662, 9636, 0), "Pull the final lever in the south-west, near the gate.");
```

Comment:
I would like linepoints for this lever too, since you have to walk past a trap. it wasn't a long walk so not very bothered about it